src/App.js

```diff
@@ -48,33 +40,18 @@ class App extends Component {
         })
     }
 
-
-    // Old plan - On click of an image, grab the ID of the current image, loop through the descriptions array (seperate)
-    // to see if id's match. If it does, update the state with that description.
-    // Cons - annoying to handle with current knowledge
-    // Pros - small storage size
-
-    // New plan - On click of image, grab the desc property (most likely undefined)
-    // on close, add a desc prop to that object in the apiData array. Store that in local storage.
-    // Cons - takes up more local storage
-    // Pros - easier to handle
-
-
-    // In local storage, have an object with keys being the id's and the values being the description
     open = (d) => {
```

Comment:
Since `open` is being used in callback form within the render function, it no longer needs to be a class property.  Doesn't _really_ matter in this case since there is only one "App".